diff -ur rpm-4.1.orig/platform.in rpm-4.1/platform.in
--- rpm-4.1.orig/platform.in	2004-12-25 20:54:41.000000000 +0200
+++ rpm-4.1/platform.in	2004-12-25 21:11:09.000000000 +0200
@@ -150,12 +150,20 @@
 @los_angeles@%vendor		Los Angeles GNU/Linux Team
 @los_angeles@
 @los_angeles@# Manpage directories.
-@los_angeles@%_man1dir		%_mandir/man1
-@los_angeles@%_man2dir		%_mandir/man2
-@los_angeles@%_man3dir		%_mandir/man3
-@los_angeles@%_man4dir		%_mandir/man4
-@los_angeles@%_man5dir		%_mandir/man5
-@los_angeles@%_man6dir		%_mandir/man6
-@los_angeles@%_man7dir		%_mandir/man7
-@los_angeles@%_man8dir		%_mandir/man8
-@los_angeles@%_man9dir		%_mandir/man9
+@los_angeles@%_man1dir		%{_mandir}/man1
+@los_angeles@%_man2dir		%{_mandir}/man2
+@los_angeles@%_man3dir		%{_mandir}/man3
+@los_angeles@%_man4dir		%{_mandir}/man4
+@los_angeles@%_man5dir		%{_mandir}/man5
+@los_angeles@%_man6dir		%{_mandir}/man6
+@los_angeles@%_man7dir		%{_mandir}/man7
+@los_angeles@%_man8dir		%{_mandir}/man8
+@los_angeles@%_man9dir		%{_mandir}/man9
+@los_angeles@
+@los_angeles@# X11 directories.
+@los_angeles@%_x11dir		%{_prefix}/X11R6
+@los_angeles@%_x11bindir	%{_x11dir}/bin
+@los_angeles@%_x11libdir	%{_x11dir}/lib
+@los_angeles@%_x11includedir	%{_x11dir}/include
+@los_angeles@%_x11mandir	%{_x11dir}/man
+@los_angeles@%_x11datadir	%{_x11dir}/share
