diff -ur hdparm-5.7.orig/Makefile hdparm-5.7/Makefile
--- hdparm-5.7.orig/Makefile	2004-12-16 04:14:26.000000000 +0200
+++ hdparm-5.7/Makefile	2004-12-16 04:14:45.000000000 +0200
@@ -10,9 +10,7 @@
 mandir = $(manprefix)/share/man
 oldmandir = $(manprefix)/man
 
-ifndef CC
 CC = gcc
-endif
 CFLAGS := -W -Wbad-function-cast -Wcast-align -Wpointer-arith -Wcast-qual -Wshadow -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -fkeep-inline-functions -Wwrite-strings -Waggregate-return -Wnested-externs -Wtrigraphs $(RPM_OPT_FLAGS)
 
 
